src/components/Visualizations/Table/HeroImage.jsx

```diff
@@ -402,4 +402,27 @@ Mmr.propTypes = {
   number: PropTypes.number,
 };
 
+export const CompetitiveRank = ({ rankTier }) => (
+  <span>
+    <section
+      data-hint="Competitive rank"
```

Comment:
remove this string or move it to lang file?  Not sure what data-hint is used for